app/src/androidTest/java/com/google/android/perftesting/PerfTestTemplate.java

```diff
@@ -50,13 +50,13 @@
 
     public EnableTestTracing mEnableTestTracing = new EnableTestTracing();
 
-    public EnablePostTestDumpsys mEnablePostTestDumpsys = new EnablePostTestDumpsys();
+    public EnablePostTestDumpsys mEnablePostTestDumpsys = new EnablePostTestDumpsys(10);
 
     public EnableLogcatDump mEnableLogcatDump = new EnableLogcatDump();
 
     public EnableNetStatsDump mEnableNetStatsDump = new EnableNetStatsDump();
 
-    public GetExecutionTime mGetExecutionTime = new GetExecutionTime();
+    public GetExecutionTime mGetExecutionTime = new GetExecutionTime(4000);
```

Comment:
How about renaming `GetExecutionTime` to `MeasureExecutionTime`? The latter seems more intuitive if a threshold is required.